apps/desktop/src/services/AgentService/infrastructure/providers/adapters/mimo.ts

```
@@ -50,7 +50,9 @@ function buildGatewayErrorDetails(
     return {
         gatewayCode: code ?? null,
         statusCode,
-        requiresRelogin: code ? reloginRequiredGatewayCodes.has(code) : false,
+        requiresRelogin: code
+            ? reloginRequiredGatewayCodes.has(code)
+            : statusCode === 401,
```

ESLint

[error] Replace `⏎············?·reloginRequiredGatewayCodes.has(code)⏎···········` with `·?·reloginRequiredGatewayCodes.has(code)`

(prettier/prettier)